dashboard/app/controllers/home_controller.rb

```diff
@@ -64,4 +84,26 @@ def terms_and_privacy
   def teacher_announcements
     render template: 'api/teacher_announcement', layout: false
   end
+
+  def recent_courses
+    current_user.in_progress_and_completed_scripts.map do |script|
+      script_id = script[:script_id]
+      script_name = Script.find_by_id(script_id)[:name]
```

Comment:
This is not hitting our cache. Is there a reason that it needs to bypass the cache?